samples/Makefile

```diff
@@ -90,7 +90,7 @@ LDFLAGS_tgidemo_atarixl    = --start-addr 0x4000
 %: %.s
 
 .c.o:
-	$(CC) $(CFLAGS) -Oirs --codesize 500 -T -g -t $(SYS) $<
+	$(CC) $(CFLAGS) -Oir --codesize 500 -T -g -t $(SYS) $<
```

Comment:
As far as I understand `-Oi` equals `--codesize 200` so if there's a `--codesize 500` following we should rather drop `-Oi`, shouldn't we?